test/qunit/tests.js

```diff
@@ -830,3 +830,18 @@ QUnit.test('null values', (assert) => {
   })
   assert.ok(swal.isVisible())
 })
+
+QUnit.test('backdrop accepts string css color', (assert) => {
```

Comment:
@acupajoe Can you update this test to reflect that users can set a `background` (not just a `background-color`)?

Reply:
Ah yes, thank you!